_delphi_utils_python/delphi_utils/weekday.py

```diff
@@ -93,11 +102,12 @@ def _fit(X, scales, npnums, npdenoms):
         for scale in scales:
             try:
                 prob = cp.Problem(cp.Minimize((-ll + lmbda * penalty) / scale))
-                _ = prob.solve()
+                _ = prob.solve(solver = cp.ECOS)
```

Comment:
is this the default solver?  i presume youre just making the choice explicit...

Reply:
Yes, it was the default until recently. This code is still using the old default, and I'm including the explicit restriction to maintain the same behavior. (ECOS isn't the source of the problems, when I was running locally it was using ECOS as wel).